addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -13483,7 +13483,50 @@ QUnit.module("Views", (hooks) => {
                         uid: 7,
                     });
                     assert.strictEqual(resModel, "partner");
-                    assert.deepEqual([...resIds], []);
+                    assert.deepEqual([...resIds], [1]);
+                },
+            });
+
+            const cpButtons = getVisibleButtons(target);
+            assert.deepEqual(
+                [...cpButtons].map((button) => button.textContent.trim()),
+                ["New", "display", ""]
+            );
+            assert.hasClass(cpButtons[1], "display");
+
+            await click(cpButtons[1]);
+            assert.verifySteps(["execute_action"]);
+        }
+    );
+
+    QUnit.test(
+        "kanban view: action button in controlPanel with multiple record in kanban",
+        async (assert) => {
+            const domain = [["id", "in", [1, 2, 3]]];
+            const kanban = await makeView({
+                type: "kanban",
+                resModel: "partner",
+                serverData,
+                arch: `
+                <kanban class="o_kanban_test">
+                    <header>
+                        <button name="display" type="object" class="display" string="display" display="always"/>
+                    </header>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div>
+                                <field name="foo"/>
+                            </div>
+                        </t>
+                    </templates>
+                </kanban>`,
+                domain,
+            });
+            patchWithCleanup(kanban.env.services.action, {
+                doActionButton: async (params) => {
+                    const { resIds } = params;
+                    assert.step("execute_action");
+                    assert.deepEqual([...resIds], [1, 2, 3]);
```

Comment:
```suggestion
                    assert.step("execute_action");
                    assert.deepEqual(params.resIds, [1, 2, 3]);
```
- Same as above. :thinking: 
- No need to unpack if you only need 1 key :slightly_smiling_face: